applications/serial_lte_modem/Kconfig

```diff
@@ -68,6 +68,12 @@ config SLM_UART_TX_BUF_SIZE
 	help
 	  Amount of UART traffic waiting to be sent (TX), that can be held. If the buffers are full, will send synchronously.
 
+config SLM_UART_ECHO
```

Comment:
How does this deal with backspace?

Reply:
Now it clears the line before putting buffer on it.